share/availability/light/availability_test.go

```diff
@@ -173,20 +154,20 @@ func BenchmarkService_GetSharesByNamespace(b *testing.B) {
 	for _, tt := range tests {
 		b.Run(strconv.Itoa(tt.amountShares), func(b *testing.B) {
 			t := &testing.T{}
-			serv, root := RandServiceWithSquare(t, tt.amountShares)
+			getter, _, root := RandServiceWithSquare(t, tt.amountShares)
```

Comment:
Do we have to return the second param? I see it's not being used in multiple places.

Reply:
I fixed this and renamed a few test methods to better match what they really return